.github/workflows/android_release.yml

```
@@ -141,26 +141,45 @@ jobs:
+        name: release-aab
+        path: android/app/build/outputs/bundle/release/app-release.aab
 
     - name: Set versionName to environment
       run: |
         VERSION_NAME=$(grep '^version_name=' android/gradle.properties | cut -d'=' -f2)
+        if [ -z "$VERSION_NAME" ]; then
+          echo "version_name not found in android/gradle.properties"
+        fi
         echo "VERSION_NAME=$VERSION_NAME" >> $GITHUB_ENV
+        echo "Version: $VERSION_NAME"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)